### PR TITLE
OAS-5912 | rebranding changes

### PR DESCRIPTION
**Pull request by adoi**

### CODE REVIEW

**Review by osilva — APPROVED** [review]
LGTM

File: docs/guides/getting-started.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ description: |-
 ---
 
 # Getting started with Oasis Terraform Provider
-ArangoDB Cloud (Oasis), provides ArangoDB databases as a Service (DBaaS). It enables you to use the entire functionality of an ArangoDB deployment without the need to run or manage the system yourself.
+ArangoGraph Insights Platform (formerly called Oasis), provides ArangoDB databases as a Service (DBaaS). It enables you to use the entire functionality of an ArangoDB deployment without the need to run or manage the system yourself.
 
-Terraform Provider Oasis is a plugin for Terraform that allows for the full lifecycle management of ArangoDB Cloud (Oasis) resources.
+Terraform Provider Oasis is a plugin for Terraform that allows for the full lifecycle management of ArangoGraph Insights Platform (formerly called Oasis) resources.
```
Comment (Robert-Stam) [inline docs/guides/getting-started.md:10]: (formerly called Oasis) can be dropped here

File: docs/guides/getting-started.md [hunk 2]
```diff
@@ -5,15 +5,15 @@ description: |-
...
 
 ## Provider Setup
 
 
 You need to supply proper credentials to the provider before it can be used. API keys serve as the credentials to the provider. You can obtain the keys from the Oasis dashboard.
-Log in to the Oasis dashboard and open the [**API Keys**](https://cloud.arangodb.com/dashboard/user/api-keys) tab of your user account. Click the **New API key** button to generate a new key, which can be used with ArangoDB's public API.
+Log in to the ArangoGraph Insights Platform (formerly called Oasis) dashboard and open the [**API Keys**](https://cloud.arangodb.com/dashboard/user/api-keys) tab of your user account. Click the **New API key** button to generate a new key, which can be used with ArangoDB's public API.
```
Comment (Robert-Stam) [inline docs/guides/getting-started.md:16]: (formerly called Oasis) can be dropped here

File: docs/guides/setup.md [hunk 3]
```diff
@@ -20,22 +20,22 @@ terraform {
 }
 
 provider "oasis" {
-  api_key_id     = "" // API Key ID generated in Oasis platform
-  api_key_secret = "" // API Key Secret generated in Oasis platform
+  api_key_id     = "" // API Key ID generated in ArangoGraph Insights Platform (formerly called Oasis) platform
+  api_key_secret = "" // API Key Secret generated in ArangoGraph Insights Platform (formerly called Oasis) platform
```
Comment (Robert-Stam) [inline docs/guides/setup.md:24]: (formerly called Oasis) can be dropped here and second platform as well

File: docs/guides/setup.md [hunk 4]
```diff
@@ -20,22 +20,22 @@ terraform {
...
 The provider can also be setup with a default organization and project to manage resources in:
 
 ```hcl
 provider "oasis" {
-  api_key_id     = "" // API Key ID generated in Oasis platform
-  api_key_secret = "" // API Key Secret generated in Oasis platform
-  organization   = "" // Oasis Organization ID
+  api_key_id     = "" // API Key ID generated in ArangoGraph Insights Platform (formerly called Oasis) platform
+  api_key_secret = "" // API Key Secret generated in ArangoGraph Insights Platform (formerly called Oasis) platform
+  organization   = "" // Organization ID within ArangoGraph Insights Platform (formerly called Oasis)
```
Comment (Robert-Stam) [inline docs/guides/setup.md:34]: (formerly called Oasis) can be dropped here, and second platform as well

File: docs/guides/setup.md [hunk 5]
```diff
@@ -20,22 +20,22 @@ terraform {
...
   project        = "" // Project ID within the specified organization
 }
 ```
 
 The other options you can provide are:
 - `oasis_endpoint` for the endpoint you want to manage the resources in (by default set to: `api.cloud.arangodb.com`).
-- `api_port_suffix` for the Oasis API Port Suffix (by default set to `:443`).
\ No newline at end of file
+- `api_port_suffix` for the ArangoGraph Insights Platform (formerly called Oasis) API Port Suffix (by default set to `:443`).
```
Comment (Robert-Stam) [inline docs/guides/setup.md:41]: (formerly called Oasis) can be dropped here

File: templates/guides/getting-started.md.tmpl [hunk 6]
```diff
@@ -5,15 +5,15 @@ description: |-
 ---
 
 # Getting started with Oasis Terraform Provider
-ArangoDB Cloud (Oasis), provides ArangoDB databases as a Service (DBaaS). It enables you to use the entire functionality of an ArangoDB deployment without the need to run or manage the system yourself.
+ArangoGraph Insights Platform (formerly called Oasis), provides ArangoDB databases as a Service (DBaaS). It enables you to use the entire functionality of an ArangoDB deployment without the need to run or manage the system yourself.
 
-Terraform Provider Oasis is a plugin for Terraform that allows for the full lifecycle management of ArangoDB Cloud (Oasis) resources.
+Terraform Provider Oasis is a plugin for Terraform that allows for the full lifecycle management of ArangoGraph Insights Platform (formerly called Oasis) resources.
```
Comment (Robert-Stam) [inline templates/guides/getting-started.md.tmpl:10]: (formerly called Oasis) can be dropped here

File: templates/guides/getting-started.md.tmpl [hunk 7]
```diff
@@ -5,15 +5,15 @@ description: |-
...
 
 ## Provider Setup
 
 
 You need to supply proper credentials to the provider before it can be used. API keys serve as the credentials to the provider. You can obtain the keys from the Oasis dashboard.
-Log in to the Oasis dashboard and open the [**API Keys**](https://cloud.arangodb.com/dashboard/user/api-keys) tab of your user account. Click the **New API key** button to generate a new key, which can be used with ArangoDB's public API.
+Log in to the ArangoGraph Insights Platform (formerly called Oasis) dashboard and open the [**API Keys**](https://cloud.arangodb.com/dashboard/user/api-keys) tab of your user account. Click the **New API key** button to generate a new key, which can be used with ArangoDB's public API.
```
Comment (Robert-Stam) [inline templates/guides/getting-started.md.tmpl:16]: (formerly called Oasis) can be dropped here

File: templates/guides/setup.md.tmpl [hunk 8]
```diff
@@ -20,22 +20,22 @@ terraform {
 }
 
 provider "oasis" {
-  api_key_id     = "" // API Key ID generated in Oasis platform
-  api_key_secret = "" // API Key Secret generated in Oasis platform
+  api_key_id     = "" // API Key ID generated in ArangoGraph Insights Platform (formerly called Oasis) platform
+  api_key_secret = "" // API Key Secret generated in ArangoGraph Insights Platform (formerly called Oasis) platform
```
Comment (Robert-Stam) [inline templates/guides/setup.md.tmpl:24]: (formerly called Oasis) can be dropped here and second platform as well

File: templates/guides/setup.md.tmpl [hunk 9]
```diff
@@ -20,22 +20,22 @@ terraform {
...
 The provider can also be setup with a default organization and project to manage resources in:
 
 ```hcl
 provider "oasis" {
-  api_key_id     = "" // API Key ID generated in Oasis platform
-  api_key_secret = "" // API Key Secret generated in Oasis platform
-  organization   = "" // Oasis Organization ID
+  api_key_id     = "" // API Key ID generated in ArangoGraph Insights Platform (formerly called Oasis) platform
+  api_key_secret = "" // API Key Secret generated in ArangoGraph Insights Platform (formerly called Oasis) platform
+  organization   = "" // Organization ID within ArangoGraph Insights Platform (formerly called Oasis)
```
Comment (Robert-Stam) [inline templates/guides/setup.md.tmpl:34]: (formerly called Oasis) can be dropped here and second platform as well

File: templates/guides/setup.md.tmpl [hunk 10]
```diff
@@ -20,22 +20,22 @@ terraform {
...
   project        = "" // Project ID within the specified organization
 }
 ```
 
 The other options you can provide are:
 - `oasis_endpoint` for the endpoint you want to manage the resources in (by default set to: `api.cloud.arangodb.com`).
-- `api_port_suffix` for the Oasis API Port Suffix (by default set to `:443`).
\ No newline at end of file
+- `api_port_suffix` for the ArangoGraph Insights Platform (formerly called Oasis) API Port Suffix (by default set to `:443`).
```
Comment (Robert-Stam) [inline templates/guides/setup.md.tmpl:41]: (formerly called Oasis) can be dropped here and second platform as well

**Review by Robert-Stam — COMMENTED** [review]
See comments

**Pull request merged**